src/appshell/qml/dockwindow/DockPanelTab.qml

```diff
@@ -56,6 +56,10 @@ StyledTabButton {
         }
 
         MenuButton {
+
+            height: 20
+            width: 20
```

Comment:
Better to use bindings when you want to set constant values for both width and height. Like this:

height: 20
width: height

Reply:
Good idea. I've made that change across all three changed files.